subtests/docker_cli/run_volumes_selinux/run_volumes_selinux.py

```diff
@@ -174,12 +162,14 @@ def check_all_files(self, volume, exp_files):
         :param volume: path to shared volume dir (on host)
         :param exp_files: expected files (relative to volume)
         """
+        # Make sure disk matches pagecache
+        utils.run("sync", 10)
         exp_files = set('%s/%s' % (volume, _) for _ in exp_files)
         act_files = set()
         for pwd, _, filenames in os.walk(volume):
             for filename in filenames:
                 act_files.add('%s/%s' % (pwd, filename))
-        self.failif(exp_files.symmetric_difference(act_files), "Not all files"
+        self.failif(exp_files.symmetric_difference(act_files), "Not all files "
```

Comment:
Very minor nit: my habit is to put the space on the continuation line, not the continued line. This makes it very obvious to the reader that the string is continued from the previous line. Again, minor.